### ISSUE TITLE
[Feature request] Sentences from a EPWING dictionary

### ISSUE THREAD

**Issue by furukasa-marui** [FooSoft/yomichan#63]
Just installed this amazing extension and I already have requests to do XD.
Could you add the ability to choose between the sentence that the extension takes from the page and the ones from the dictionary? The main reason is because most of the time I get small sentences.

Also, using 新和英大辞典(第5版) I get the whole page, with the definition, sentences, translation and other uses. Separating the contents would be very useful (at least for me) and would take less time for me to edit my template.

**Comment by FooSoft** [comment]
Glad you like the extension!

What do you mean when you say "ones from the dictionary" regarding sentences? Are you talking about JMDICT or something that's in 新和英大辞? I would love to be able to split the dictionary entries into more logical chunks, but it is really hard to parse because the entry items are not separated at all in the EPWING format.

**Comment by furukasa-marui** [comment]
I'm talking about the sentences from 新和英大辞. That's is bad :(. I it was possible because when I used Epwing2Anki it recognized the sentences, definition and translation. So, can you add an option to copy the words (maybe with a combination of buttons?) to an external ".txt" file?
